MekHQ/src/mekhq/campaign/personnel/education/EducationController.java

```diff
@@ -314,19 +314,19 @@ public static boolean processNewDay(Campaign campaign, Person person, boolean ag
             return false;
         }
 
-        // is person on campus and undergoing education?
-        if (educationStage == 2) {
+        // is person on campus and undergoing education, or awaiting assignment to a sibko?
+        if ((educationStage == 2) || (educationStage == 3)) {
```

Comment:
This would ideally be an enum like EducationStage.ON_CAMPUS, EducationStage.AWAITING_SIBKO

Reply:
Oh, good call! That never even occurred to me. I'll see it done